doc/source/placement-group.rst

```diff
@@ -0,0 +1,140 @@
+Placement Group
+===============
+
+Ray is great at scheduling tasks and actors based on its resource requirements.
+requirements, it is also important to specify which "place" they should be scheduled. Ray's placement group APIs allow you to place your actors or tasks according to your strategies.
+
+Let's see some scenarios it is useful.
+
+- **Gang scheduling**: Let's say you'd like to place exact number of actors on each GPU node to maximize bandwidth.
```

Comment:
This isn’t gang scheduling right? This is colocation?

Reply:
Yes, I think we should make it clear that we want all of the actors to run or none of them.